ovs/lib/storagedriver.py

```diff
@@ -399,6 +408,200 @@ def _configure_arakoon_to_volumedriver(cluster_name):
                                                                           dls_arakoon_cluster_nodes=arakoon_nodes)
                     storagedriver_config.save()
 
+    @staticmethod
+    def create_new_storagedriver(vpool_guid, storagerouter_guid, storage_ip, amount_of_proxies):
```

Comment:
Tautology detected + wouldn't it be niftier to create some sort of StorageDriverConfigurator / StorageDriverInstaller class which can be initiated with a vpool guid, sr guid and such and then add proxies to it, configure stuff, ...
Then we don't need to pass all the information through dicts and many args and kwargs